src/joyaxis.cpp

```diff
@@ -295,6 +295,23 @@ int JoyAxis::calculateThrottledValue(int value)
     return temp;
 }
 
+/**
+ * @brief Checks if the axis supports haptic feedback.
+ * @returns True if the axis supports haptic feedback, false otherwise.
+ */
+bool JoyAxis::hasHapticFeedback() const { return false; }
+
+/**
+ * @brief Always returns nullptr for JoyAxis base objects.
+ */
+HapticFeedback *JoyAxis::getHapticFeedback() const { return nullptr; }
+
+/**
+ * @brief Changes the haptic feedback effect mode.
+ * @param[in] mode New haptic feedback effect mode.
+ */
+void JoyAxis::setHapticFeedbackMode(HapticFeedbackMode) {}
+
```

Comment:
Just like above.  
A bit too general name. Methods' descriptions are good, but names like `hasHapticFeedback`suggest more generic haptic feedback (like classic rumble).

Reply:
I changed it to hasHapticTrigger.